File: ITSense/BodegaAPI/BodegaAPI/BodegaAPI/Data/Entities/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BodegaAPI.Data.Entities
{
    public class Product
    {
        public Product()
        {

        }
        public Guid Id { get; set; }
        public string Name { get; set; }
        public int Status { get; set; }
        public bool Defective { get; set; }
    }
}

File: ITSense/BodegaUI/src/app/Modules/Products/Models/Product.ts
import { Guid } from "guid-typescript";
import { EMPTY } from "rxjs";

export class Product{
  id: Guid;
  name: string;
  status:boolean;


constructor() {
  this.id= Guid.createEmpty();
  this.name='';
  this.status=false;
}
}
